Makefile

```diff
@@ -4,9 +4,8 @@
 
 PYTHON ?= python
 CYTHON ?= cython
-NOSETESTS ?= nosetests
-NOSETESTS_OPTIONS := $(shell pip list | grep nose-timer > /dev/null && \
-                       echo '--with-timer --timer-top-n 50')
+TEST_RUNNER ?= pytest
+TEST_RUNNER_OPTIONS := --durations=0 -vv
```

Comment:
TEST_RUNNER_OPTIONS doesn't seem to be used anywhere?

Reply:
I guess you can just remove it if you have moved those options to pytest.ini

Reply:
Yeah

Reply:
I removed the TEST_RUNNER env var as well. Redundant.